addons/playersuppression/functions/fnc_fired.sqf

```diff
@@ -0,0 +1,26 @@
+/*
+ * Author: LAxemann
+ * initial checks, gets config values etc, input from the fired EH
+ */
+#include "script_component.hpp"
+private ["_unit"];
+_unit		= _this select 0;         // Unit that shot the projectile
+if ((player distance _unit ) >= 2700) exitWith {};
+if ((side effectiveCommander (vehicle _unit)) != (side player)) then {
+    private ["_ammo", "_weapon", "_projectile", "_config", "_hit", "_dDist"];
+    _weapon     = _this select 1;     // Weapon fired
+    if ((_weapon in ["throw","put"])) exitWith {};

```

Comment:
This doesn't work. STRING in ARRAY is case sensitive. By default, Throw and Put are captitalized, but any shitty 3rd party addon could change this by simple inheritance.
use:
`toLower _weapon in ["throw","put"]`